### PR TITLE
README + env variable naming

### PR DESCRIPTION
**Pull request by jonmattgray**
- Update the README to include the usual sections + some explanation of binaries.
- host/address consistency in env variables
- change default QEMU port to `1022` to match docker compose port

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,55 @@
...
+
+## Configuration
+
+Use a `.env` at root of the repository to set values for the environment variables defined in [custom-environment-variables.json](./config/custom-environment-variables.json).
+
+| variable         | required |   default   | description                                                                          |
+| :--------------- | :------: | :---------: | :----------------------------------------------------------------------------------- |
+| PORT             |    N     |   `3000`    | The port for the API to listen on                                                    |
+| LOG_LEVEL        |    N     |   `debug`   | Logging level. Valid values are [`trace`, `debug`, `info`, `warn`, `error`, `fatal`] |
+| MORELLO_HOST     |    N     | `127.0.0.1` | Morello SDK QEMU host name                                                           |
```
Comment (dblane-digicatapult) [inline README.md]: Not necessarily QEMU hostname.  Just "Morello hostname" is fine here

File: README.md [hunk 2]
```diff
@@ -1,29 +1,55 @@
...
+| MORELLO_PORT     |    N     |   `1022`    | Morello SDK QEMU port                                                                |
```
Comment (dblane-digicatapult) [inline README.md]: Same

File: README.md [hunk 3]
```diff
@@ -1,29 +1,55 @@
...
+| MORELLO_USERNAME |    N     |   `root`    | Morello SDK QEMU username                                                            |
```
Comment (dblane-digicatapult) [inline README.md]: Same

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jonmattgray) [reply README.md]: done and removed references to QEMU in couple of other places

File: src/controllers/scenario/index.ts [hunk 4]
```diff
@@ -1,11 +1,4 @@
-import {
-  Controller,
-  Get,
-  Query,
-  Path,
-  Route,
-  Response,
-} from 'tsoa'
+import { Controller, Get, Query, Path, Route, Response } from 'tsoa'
```
Comment (mattdean-digicatapult) [inline src/controllers/scenario/index.ts]: hang on I'm confused. Why wasn't this failing linting before?

File: README.md [hunk 5]
```diff
@@ -1,29 +1,55 @@
-# Morello-API
-The interface for executing binaries on morello board and self that returns output which can be processed by the client.
+# morello-api
 
+## Description
```
Comment (mattdean-digicatapult) [inline README.md]: I'm not sure why but if you go "view file" this isn't rendering on github correctly

File: README.md [hunk 6]
```diff
@@ -1,29 +1,55 @@
...
+| :--------------- | :------: | :---------: | :----------------------------------------------------------------------------------- |
+| PORT             |    N     |   `3000`    | The port for the API to listen on                                                    |
+| LOG_LEVEL        |    N     |   `debug`   | Logging level. Valid values are [`trace`, `debug`, `info`, `warn`, `error`, `fatal`] |
+| MORELLO_HOST     |    N     | `127.0.0.1` | Morello host name                                                                    |
+| MORELLO_PORT     |    N     |   `1022`    | Morello port                                                                         |
+| MORELLO_USERNAME |    N     |   `root`    | Morello username                                                                     |
+
+[default.json](./config/default.json) contains default configuration values.
+
+Alternatively create a `./config/local.json` file to overwrite the values of `default.json`.
```
Comment (mattdean-digicatapult) [inline README.md]: `override` not `overwrite`

File: README.md (hunk 5 above)
Comment (jonmattgray) [reply README.md]: good spot! There was something wrong with the whitespace character?

File: README.md (hunk 6 above)
Comment (jonmattgray) [reply README.md]: changed

File: src/controllers/scenario/index.ts (hunk 4 above)
Comment (jonmattgray) [reply src/controllers/scenario/index.ts]: `"lint": "eslint ./src"` in package scripts  is the wrong command. I think it should be `eslint .` and then we add certain directories like `build` to `.eslintignore`. Will do in a separate PR, there's loads of linting errors

Comment (jonmattgray) [reply src/controllers/scenario/index.ts]: https://digicatapult.atlassian.net/browse/DSBD-72

**Review by mattdean-digicatapult — APPROVED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**